jina/drivers/cache.py

```diff
@@ -38,16 +38,16 @@ def _apply_all(self, docs: 'DocumentSet', *args, **kwargs) -> None:
                 else:
                     self.on_miss(d, data)
 
-    def on_miss(self, req_doc: 'Document', data: Any) -> None:
+    def on_miss(self, req_doc: 'Document', value: str) -> None:
         """Function to call when document is missing, the default behavior is to add to cache when miss.
 
         :param req_doc: the document in the request but missed in the cache
-        :param data: the data besides the `req_doc.id` to be passed through to the executors
+        :param value: the data besides the `req_doc.id` to be passed through to the executors
         """
         if self.with_serialization:
-            self.exec_fn(req_doc.id, req_doc.SerializeToString(), **{DATA_FIELD: data})
+            self.exec_fn([req_doc.id], req_doc.SerializeToString(), [value])
```

Comment:
Codecov said this line is not covered by tests. Not sure if Codecov is correct.

Reply:
That's correct. For now, we don't have a cache using this code. But you are right it should be tested in the future.